hs/Development_Config.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+{-
```

Comment:
@akbiggs Didn't notice earlier, but if you want Haddock to know about this, you need to use `{-|`